.github/workflows/php-test-plugins.yml

```diff
@@ -71,6 +71,6 @@ jobs:
       - name: Composer update
         run: npm run wp-env run tests-cli -- --env-cwd="wp-content/plugins/$(basename $(pwd))" composer update --no-interaction
       - name: Running single site unit tests for plugins
-        run: npm run test-php-plugins
```

Comment:
As you removed `multisite.xml`, Could you remove `tests/multisite.xml`?

Reply:
Did that in https://github.com/WordPress/performance/pull/1065/commits/bd66bb964b629a07c0bb83036f4bd7162187ac3e

Reply:
I mean we should remove that file reference from workflow file.

Reply:
Ahh okay. Done in https://github.com/WordPress/performance/pull/1065/commits/61eb09297b58a19c373ca2381538a5a9291452c1.